src/www/system_certmanager.php

```diff
@@ -1342,18 +1342,16 @@ function internalca_change() {
                     <td style="width:78%">
                       <script type="text/javascript">
                       function basic_constraints_enabled_sign_csr_refresh() {
-                          basic_constraints_enabled = $('#basic_constraints_enabled_sign_csr').prop('checked');
-                          $('#basic_constraints_is_ca_sign_csr').prop('disabled', !basic_constraints_enabled)
+                          basic_constraints_enabled = $('#basic_constraints_is_ca_sign_csr').prop('checked');
                           $('#basic_constraints_path_len_sign_csr').prop('disabled', !basic_constraints_enabled)
                       }
                       </script>
-                      <input type="checkbox" name="basic_constraints_enabled_sign_csr"  id="basic_constraints_enabled_sign_csr"  value="true" onchange="basic_constraints_enabled_sign_csr_refresh();" /> <?= gettext('basicConstraints enabled'); ?><br />
-                      <input type="checkbox" name="basic_constraints_is_ca_sign_csr"    id="basic_constraints_is_ca_sign_csr"    value="true" /> <?= gettext('is CA'); ?><br />
+                      <input type="checkbox" name="basic_constraints_is_ca_sign_csr"    id="basic_constraints_is_ca_sign_csr" onchange="basic_constraints_enabled_sign_csr_refresh();" value="true" /> <?= gettext('is CA'); ?><br />
                       <?= gettext('Path Len'); ?>: <input type="text"     name="basic_constraints_path_len_sign_csr" id="basic_constraints_path_len_sign_csr" size="5" value="<?=$pconfig['basic_constraints_sign_csr'];?>"/>
```

Comment:
This should say ```Path Length``` and explain what it is about, see https://blogs.technet.microsoft.com/pki/2014/03/05/constraints-what-they-are-and-how-theyre-used/ and http://www.pkiglobe.org/index.html (I like their wording: ```Path length gives the maximum number of non-self-issued intermediate certificates that may follow this certificate in a valid certification path.```).

Reply:
This wording seems to have come from RFC5280 section 4.2.1.9, so it seems to be good to use it.

https://tools.ietf.org/html/rfc5280#section-4.2.1.9